src/components/modal/AposModalFooter.vue

```diff
@@ -18,5 +18,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
+  .apos-modal__footer {
+    grid-row: 3 / 4;
```

Comment:
Not critical, but seems good to be explicit.

Reply:
This wasn't showing up at first as I had the style in the AposModal component. It took a bit for me to realize that if the class didn't exist in that specific component then it wouldn't work (probably because these are scoped styles).